tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -57,5 +59,27 @@ public async Task TestStaticFileConverter()
             Assert.IsTrue(bytes.Length == new FileInfo(fullPath).Length);
             Assert.IsTrue(fi.RelativePath.Equals(Path.Combine("wwwroot", _testStaticFilePath)));
         }
+
+        [Test]
+        public async Task TestWebConfigFileConverter()
+        {
+            FileConverter fc = new ConfigFileConverter(_testWebConfigFilePath);
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.First();
+            byte[] bytes = fi.FileBytes;
+
+            string resPath = Path.Combine(_testAreaFullPath, "appsettings.json");
+            string fullTestPath = Path.Combine(_testAreaFullPath, "Test_appsettings.json");
+
+            var appSettingsContent = File.ReadAllText(resPath);
+
+            Assert.True(appSettingsContent.Contains("UseMockData"));
+            Assert.True(appSettingsContent.Contains("UseCustomizationData"));
+            Assert.True(appSettingsContent.Contains("CatalogDBContext"));
+
+            //Not valid since different appsettings.json files can both work
+            //Assert.IsTrue(bytes.Length == new FileInfo(fullTestPath).Length);
+            File.Delete(resPath);
```

Comment:
Similar as above, need to find a safe way to delete this in case test fails. Maybe wrap entire test in a `try` block and delete the expected file in the `finally` block (if the file exists).